sdcm/gemini_thread.py

```diff
@@ -143,11 +143,11 @@ def _generate_gemini_command(self):
 
         stress_cmd = self.stress_cmd.replace("\n", " ").strip()
 
-        for key, value in self.gemini_default_flags.items():
-            if not key in stress_cmd:
-                cmd += f"--{key}={value} "
+        cmd = " ".join(
```

Comment:
`cmd = " ".join(f"--{key}={value}" for key, value in self.gemini_default_flags.items() if key not in stress_cmd) + stress_cmd`

I would use something like this

Reply:
already fixed

Reply:
ok